10.9-libcxx/stable/main/finkinfo/graphics/libavcodec60-6.0-shlibs.info

```diff
@@ -110,9 +111,12 @@ Source-Checksum: SHA256(62fe9fd098cb537d4b61731b11049e1ac179f38c870a41b8e9d556af
 ###
 PatchFile: %n_texinfo72.patch
 PatchFile-MD5: 460868fa2b167e7c57f605d1018c4374
+PatchFile2: %n.patch
+PatchFile2-Checksum: SHA256(9abfedbb69b60f2d1c182a31e517d49c8d58906993488617865d995ec9394fb7)
 PatchScript: <<
 	#!/bin/sh -ev
 	%{default_script}
+	patch -p1 < %{PatchFile2}
```

Comment:
```suggestion
```
It appears this is already done by `%{default_script}`; with this I am getting a second application of `PatchFile2` and
> Reversed (or previously applied) patch detected!

failure.

Reply:
Correct. `%{default_script}` applies all listed PatchFileN from  the .info, so you don't need to manually list each one as well. If `%{default_script}` is omitted, then each PatchFile must be applied manually.

Reply:
Ah, I didn't realize that from the default script.  Will change.  I knew that patch was not in the first patch file.